build.gradle

```diff
@@ -76,6 +75,10 @@ subprojects {
         ballerinaStdLibs "io.ballerina.stdlib:io-ballerina:${stdlibIoVersion}"
         ballerinaStdLibs "io.ballerina.stdlib:sql-ballerina:${stdlibSqlVersion}"
         ballerinaStdLibs "io.ballerina.stdlib:mysql-ballerina:${stdlibMysqlVersion}"
+        externalJars (group: 'mysql', name: 'mysql-connector-java', version: "${mySqlDriverVersion}") {
+            transitive = false
+        }
```

Comment:
Do we depend on MySQL connector jar for the implementation or only at runtime.

Reply:
We use it at runtime. specifically, we used it for the command line testing and can be removed if we can copy the jar file manually for testing